port/common/j9nls.c

```diff
@@ -234,34 +229,33 @@ writeSyslog(struct OMRPortLibrary *portLibrary, uintptr_t flags, const char *for
 		return;
 	}
 
-	/* Attempt to write string to on-stack buffer */
-	COPY_VA_LIST(copyOfArgs, args);
-	numberWritten = portLibrary->str_vprintf(portLibrary, outputBuffer, sizeof(outputBuffer), format, copyOfArgs);
+	char localBuffer[256];
+	char *writeBuffer = NULL;
```

Comment:
This is not valid C code.  "For C89, you must declare all of your variables at the beginning of a scope block"